modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -47,3 +46,33 @@ reviews-v3-6dfd49b55b-vcwpf       2/2     Running   0          12m
 echo "http://$GATEWAY_URL/productpage"
 ----
 . Copy and paste the output in a web browser to verify the Bookinfo product page is deployed.
+
+.Procedure from Kiali web console
+
+. Obtain the address for the Kiali web console.
+
+.. Log in to the {product-title} web console as a user with cluster-admin rights. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.. Navigate to *Networking* -> *Routes*.
+
+.. On the *Routes* page, select the control plane project, for example `istio-system`, from the *Namespace* menu.
++
+The *Location* column displays the linked address for each route.
++
+
+.. Click the link in the *Location* column for Kiali.
+
+.. Click *Log In With OpenShift*. The Kiali *Overview* screen presents tiles for each project namespace.
+
+. In Kiali, click *Graph*.
+
+. Select bookinfo from the *Namespace* list, then select App graph.
```

Comment:
```suggestion
. Select bookinfo from the *Namespace* list, then select App graph from the *Graph Type* list.
```

Reply:
For what it's worth, there is no real need to change the graph type.  You can validate the install with the default graph type, which is "Versioned app graph".  It will show traffic but it would produce a different image than the one you have here in the PR.

Reply:
Looks like this is unchanged, bit that is OK.